main.go

```diff
@@ -25,7 +28,19 @@ func main() {
 
 	scanner := bufio.NewScanner(os.Stdin)
 	for scanner.Scan() {
-		if err := verifyToken(scanner.Text()); err != nil {
+		stdinArgs := strings.Split(scanner.Text(), "||")
+		var checkPath = false
+		if len(stdinArgs) == 1 {
+			checkPath = false
+		} else if len(stdinArgs) == 2 {
+			checkPath = true
+			requestPath = stdinArgs[1]
+		} else {
+			fmt.Fprintln(os.Stdout, "INVALID NUMBER OF STDIN ARGS SUPPLIED")
+			continue
+		}
+
+		if err := verifyToken(stdinArgs[0], checkPath); err != nil {
 			fmt.Fprintln(os.Stdout, "INVALID -", err)
```

Comment:
fmt.Errorf?

Reply:
I believe we did initially use this but due to how we want apache to handle the output from this we want it to just be a standard output